Sun Aug 13 09:41:49 2000  Anthony Green  <green@redhat.com>

	* check-init.c (check_init): Add case for BIT_FIELD_REF (required
	for -pg builds).

--- gcc/java/check-init.c	2000/06/22 05:17:33	1.23
+++ gcc/java/check-init.c	2000/08/13 16:43:15
@@ -611,6 +611,7 @@ check_init (exp, before)
     case BIT_NOT_EXPR:
     case CONVERT_EXPR:
     case COMPONENT_REF:
+    case BIT_FIELD_REF:
     case NOP_EXPR:
     case FLOAT_EXPR:
     case FIX_TRUNC_EXPR:
